_posts/2022-11-20-jump_dev_2023.md

```diff
@@ -6,11 +6,10 @@ categories: [jump-dev]
 permalink: /meetings/jumpdev2023/
 ---
 
-We are pleased to announce that JuMP-dev 2023 will be co-located with
-[JuliaCon 2023](https://juliacon.org/2023)!
+JuMP-dev 2023 was held in person, 27-29 July, 2023 in Cambridge,
+Massachusetts. It was co-located with [JuliaCon 2023](https://juliacon.org/2023).
 
-This year, JuMP-dev is being held in person, 27-29 July, 2023 in Cambridge,
-Massachusetts.
+<img src="/assets/jump-dev-workshops/jump_dev_2023_speakers.jpg" alt="Photo of Speakers and Organizers">
```

Comment:
Could make it clear that this is a subset of speakers and organizers?

Reply:
Done